aguskoll/target-app
# Location permission

### What's in

Asking for location permission
Getting the location and adding the marker in the correct location in the map. 

### How it looks

https://drive.google.com/file/d/1-ncd3tQCvQ2d4jB3dyq9OX7Bwa23gsuB/view?usp=sharing

### How to test

1. Allow the app to access your location
2. Verify that the marker is added in the correct location
3. Deny the permission
4. Verify that you don't see the marker.
